include/bluetooth/mesh/light_ctl.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+/**
+ * @file
+ * @defgroup bt_mesh_light_ctl Light CTL models
+ * @{
+ * @brief API for the Light CTL models.
+ */
+
+#ifndef BT_MESH_LIGHT_CTL_H__
+#define BT_MESH_LIGHT_CTL_H__
+
+#include <bluetooth/mesh.h>
+#include <bluetooth/mesh/model_types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Minimum permitted Temperature level */
+#define BT_MESH_LIGHT_CTL_TEMP_RANGE_MIN 800
+/** Maximum permitted Temperature level */
+#define BT_MESH_LIGHT_CTL_TEMP_RANGE_MAX 20000
+
+/** All Light CTL parameters */
+struct bt_mesh_light_ctl_params {
+	/** Light level */
+	uint16_t light;
+	/** Temprature level */
+	uint16_t temp;
+	/** Delta UV level */
+	int16_t delta_uv;
+};
+
+/** Temprature subset of Light CTL parameters. */
+struct bt_mesh_light_ctl_temp_params {
```

Comment:
I'm kind of inclined to rename `bt_mesh_light_ctl_temp` to just `bt_mesh_light_temp` throughout - model, state and all. It's not a 1-to-1 match with the spec, but it's more readable, and still unambiguous